src/docfx/template/TemplateEngine.cs

```diff
@@ -195,12 +196,18 @@ public string ProcessHtml(ErrorBuilder errors, FilePath file, string html)
         return LocalizationUtility.AddLeftToRightMarker(_cultureInfo, result);
     }
 
-    private string CreateContent(FilePath file, string? mime, JObject pageModel)
+    private string CreateContent(FilePath file, string? mime, ref JObject pageModel)
     {
-        if (JsonSchemaProvider.IsConceptual(mime) || JsonSchemaProvider.IsLandingData(mime))
+        if (JsonSchemaProvider.IsConceptual(mime))
         {
-            // Conceptual and Landing Data
-            return pageModel.Value<string>("conceptual") ?? "";
+            var conceptual = JsonUtility.ToObject<ConceptualModel>(ErrorBuilder.Null, pageModel)!;
+            ProcessConceptualHtml(pageModel.Value<string>("conceptual") ?? "", conceptual);
+            pageModel = JsonUtility.SortProperties(_config.DryRun ? new JObject() : JsonUtility.ToJObject(conceptual));
+            return conceptual.Conceptual ?? "";
```

Comment:
seems sorting is handled by [https://github.com/dotnet/docfx/blob/a4e88c1a3f974b62d44884f827e6488e2959067d/src/docfx/build/page/PageBuilder.cs#L159-L159](https://github.com/dotnet/docfx/blob/a4e88c1a3f974b62d44884f827e6488e2959067d/src/docfx/build/page/PageBuilder.cs#L159-L159) already

Reply:
This should be sorted because it will affect the order of page metadata in https://github.com/dotnet/docfx/blob/948f000506002aca42acb0635f25097628ba0d76/src/docfx/template/TemplateEngine.cs#L207.

`pageModel` was sorted before passing in this method before, but it has been changed within this method during this PR.

Reply:
Now we don't need to sort at all here xD